### ISSUE TITLE
Redesign Landing page

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#3844]
<img width="1436" alt="Screenshot 2022-08-26 at 15 56 04" src="https://user-images.githubusercontent.com/5887929/186920082-9c03b4dd-ae75-4abf-8318-aae482ccf6e8.png">

We've been working quite a while on a new design, and we will start rolling with a new one as mentioned in https://github.com/kodadot/nft-gallery/discussions/3710
This issue is in the draft, and I will gradually add descriptions of particular sections. 

@exezbcz will drop some more wisdom I've forgot and help you guide through
- mobile description coming<sup>soon</sup>

<img src="https://media1.giphy.com/media/l0HlFFTVOQz0bKYJG/giphy.gif"/>

## Figma export 
- [Readme](https://www.figma.com/file/3iOjW12ERFRDSVnpLuhQmf/landing-page-handoff?node-id=1305%3A5612) 
- [Design](https://www.figma.com/file/3iOjW12ERFRDSVnpLuhQmf/landing-page-handoff?node-id=798%3A3662)
- [Components + Prototypes](https://www.figma.com/file/3iOjW12ERFRDSVnpLuhQmf/landing-page-handoff?node-id=1305%3A5613)

## Navbar 
Navbar will be functioning most likely nowadays, with some refreshment.

### From left to right
- There will be a logo which takes you back home `/`.
- Search in the navbar, which will appear once you scroll the site.
- Explore, which will take you to global explorer -- I guess we don't have this one, so most likely skip it for now    
- Stats - Rankings like series 
- Connect -- this will stay the current one. Hopefully, it's already made as a component (?)

## Search
We will have a multi-chain search including RMRK, Basilisk and other upcoming implementations.
Multichain search will support [^supportedchain]
The search bar could be selected when pressing 'K' 
On input focus, it will show trending collections
those trending collections could be fetched by default by most volume or most of buys per 7 days with the option to overwrite/add extra collections to list
Under search, we can show currently working chains , along with upcoming integrations [^chains]
Each chain name would be linked to a particular explorer feed based on that chain. I.e. RMRK ->  `/rmrk/explore`

The same search bar would go to the navbar once the user scrolls down
The majority of search is being done in 
- #3763 
- #3360

## Spotlight 

This section would be a remake of our existing carousel component. The component will be still and without auto-play. 
On this component, we will be showing collections from all chains. So having array with chain and address collection would be mandatory here. 
As you see, we will be showing picture of collection with collection name. The link will take user to the collection view.
The spotlight will be horizontally scrollable, with at least 15 collection. 
This feed will be a curated list by community picks. 

Some work is being done here
- #3772 
- #876

## Top collections
The collection would receive a new look and feel. 
Most data are already done in the series insights component, so expanding here should be a breeze. 
The new functionality will be there, showing 24 hours, 7 days and 30 days time frame of data.
Default selected time frame should be 7 days.
On the upper right, "All" would change collection data based on supported chains: [^supportedchain].  

### Data
- name of collection
- floor of the collection 
- total volume of collection in native token per selected period
- total volume of collection in USD per selected period
- difference within the selected period

Link will take the user to the collection view.

Some part of preparation is upcoming in this issue.

- #3869 

## New Listings
This section would work same as current listings. 
The major difference is that we will pool data from various indexers into one component. 
We will be showing user on which chain listing happened, like [^supportedchain] .

### Data
- NFT name
- Collection name
- Listing price 
- Supported chains: [^supportedchain]

## Latest Sales 
This section would work same as current sales. 
The major difference is that we will pool data from various indexers into one component. 
We will be showing user on which chain sale happened, like supported chains:  [^supportedchain] .

### Data
- NFT name
- Collection name
- Sale price (the last buy happened)
- Supported chains:  [^supportedchain]

## Featured Articles
We will have a list of articles, for now, three without scroll. 

### Data
- hero picture for the article 
- Article title
- Short description
- Link to an article

More articles can go to either to Medium or Substack publications, whichever we will decide for 

## Footer 
Here we will collect email address into our Substack. 
Link Substack of our Kodadot publication, can drop code for button
Other links are to
- FAQ -> https://kodadot.xyz/rmrk/faq -> probably we should move FAQ to root, like https://kodadot.xyz/faq 
- Grants -> link to current grants form 
- Documentation -> https://docs.kodadot.xyz/
- Contribute -> https://kodadot.xyz/contribute
#### Socials 
- Twitter -> https://twitter.com/kodadot
- Instagram -> https://instagram/kodadot.xyz
- Discord -> https://discord.gg/u6ymnbz4PR 
- Youtube 
- Telegram 

## Legend

- https://github.com/kodadot/nft-gallery/issues/2792

[^chains]: Currently supported chains are RMRKv1 and Basilisk Rococo. Upcoming integrated chains will be MoonRiver, MoonBeam and Astar(RMRK) 

[^supportedchain]: RMRK 1.0, Basilisk

### Currently working chains
- RMRKv1 
- Basilisk Rococo
### Upcoming working chains
- MoonRiver
- MoonBeam
- Astar (RMRK)

**Comment by preschian** [comment]
Hi, because some ordering components are a little bit different in the new design, I will prepare a new landing-page component with a query parameter approach https://github.com/kodadot/nft-gallery/discussions/3710#discussioncomment-3501133. Hopefully, everyone can start to redesign new components in that layout. And, it would be nice if the new components were written with Vue Compose API

Ref:
- https://v3.nuxtjs.org/bridge/overview
- https://vuejs.org/guide/typescript/composition-api.html
- https://v3.nuxtjs.org/guide/directory-structure/composables/
- https://v3.nuxtjs.org/guide/features/state-management

Task:
- [ ] Redesign: prepare separate landing page

Currently, I'm still in progress redesigning the carousel for the spotlight, new listing, and the latest sales
- https://github.com/kodadot/nft-gallery/pull/3889

The other components maybe can be started:
- [ ] Redesign: navbar
- [ ] Redesign: featured articles
- [ ] Redesign: footer

**Comment by prachi00** [comment]
@preschian @yangwao can I start working on the footer design? 
<img width="702" alt="Screen Shot 2022-09-07 at 10 08 38 PM" src="https://user-images.githubusercontent.com/39299315/189039131-e920da06-6850-4fd7-a12e-b7bf398bd9c0.png">

**Comment by preschian** [comment]
> @preschian @yangwao can I start working on the footer design? 

of course, yes, I guess 👍
you can start by creating a new branch from this branch https://github.com/kodadot/nft-gallery/pull/3895/files (while this branch is still in review)
for the new footer put the component in here `components/TheFooter.vue`

**Comment by prachi00** [comment]
> > @preschian @yangwao can I start working on the footer design?
> 
> of course, yes, I guess 👍 you can start by creating a new branch from this branch https://github.com/kodadot/nft-gallery/pull/3895/files (while this branch is still in review) for the new footer put the component in here `components/TheFooter.vue`

sounds good, thank you, I'll start on this one

**Comment by yangwao** [comment]
Hey since https://github.com/kodadot/nft-gallery/pull/3895 is merged others can help with this? 👀

**Comment by Jarsen136** [comment]
I will start working on bringing searchBar to landing page once #3894 is merged

<img width="748" alt="image" src="https://user-images.githubusercontent.com/31397967/189657747-7ca0dd10-35e7-40c6-b42f-0c222eddc147.png">

**Comment by yangwao** [comment]
> I will start working on bringing searchBar to landing page once #3894 is merged

stage is yours :)

**Comment by yangwao** [comment]
let's continue to 
- #4090
